guides/release/getting-started/working-with-html-css-and-javascript.md

```diff
@@ -1,54 +1,86 @@
-Ember is a frontend framework written in _JavaScript_, the primary scripting
-language of the web. You don't need to be an expert in JavaScript to begin using
-Ember, but it helps to have some general programming knowledge as well as some
-basic knowledge of JavaScript and how it differs from other languages.
+Ember is a framework for building applications that run in the browser, which
+means that they are made with HTML, CSS, and JavaScript. You don't need to be an
+expert in any of these technologies to write an Ember app, but it's good to have
+some basic working knowledge of them in order to be able to work with Ember, and
+to work through the guides.
```

Comment:
"It is very helpful to be familiar with these technologies. If you find yourself getting stuck or confused as you learn Ember, come back to this page and see if there is a general topic below that you could explore."

Let's avoid the word expert here, because it makes it sound like "it's ok if you're not an expert! Intermediate knowledge will do."